src/include/models.h

```diff
@@ -41,59 +16,91 @@ struct FuncConstVel
             0.0, 0.0, 0.0, 0.0, 1.0, T,
             0.0, 0.0, 0.0, 0.0, 0.0, 1.0;
 
-        //------------ЭКСТРАПОЛЯЦИЯ СИГМА-ВЕКТОРОВ-----------------
-        std::vector<M> Xue(Xu.size());
+        M Xue(Xu.rows(),Xu.cols());
 
-        for (int i = 0; i < Xu.size(); i++)
+        for (int i = 0; i < Xu.cols(); i++)
         {
-            Xue[i] = F * Xu[i]; 
+            Xue.col(i) = F * Xu.col(i); 
         }
         
         return Xue;
     }
 };
 
-
-
 template <class M>
-struct FuncMeasSph
-{ 
-        M operator()(const M X, M Z)
-    {   
-        SizeMat rows = SizeMat::ROW3;
-        SizeMat cols = SizeMat::COL1;
+struct FuncConstTurn
+{
+
+    M operator()(const M &Xu, double T)
+    {
+        M F(ENUM_TO_INT(SizeMat::ROW7),ENUM_TO_INT(SizeMat::COL7));
 
-        double range = sqrt(pow(X(0, 0), 2) + pow(X(2, 0), 2) + pow(X(4, 0), 2));
-        double az = atan2(X(2, 0), X(0, 0));
-        double el = atan2(X(4, 0), sqrt(pow(X(0, 0), 2) + pow(X(2, 0), 2)));
+        M Xue(Xu.rows(),Xu.cols());
+        for (int i = 0; i < Xu.cols(); i++)
+        {
+            double w = Xu.col(i)(ENUM_TO_INT(CoordPositionMat::W));
+
+            F << 1.0,  sin(w*T)/w,      0.0,   -(1-cos(w*T))/w,    0.0,    0.0,   0.0,
+                0.0,  cos(w*T),         0.0,    -sin(w*T),          0.0,    0.0,   0.0,
+                0.0,  (1-cos(w*T))/w,   1.0,    sin(w*T)/w,         0.0,    0.0,   0.0,
+                0.0,  sin(w*T),         0.0,    cos(w*T),           0.0,    0.0,   0.0,
+                0.0,  0.0,              0.0,    0.0,               1.0,      T,   0.0,
+                0.0,  0.0,              0.0,    0.0,               0.0,    1.0,   0.0,
```

Comment:
А тут не хватает T

Reply:
https://www.mathworks.com/help/fusion/ug/motion-model-state-and-process-noise.html
на матлабовском сайте преведена модель CT
![image](https://github.com/AlekseevEst/Estimator/assets/159806597/1b367c7c-1162-44d6-a1e4-b4d08e68618a)

и написано: "To model 3-D motion, simply add the z-motion equation of the constant velocity model since the z-motion is decoupled from the x-y motion in the constant turn model." 
Рус: Чтобы смоделировать трехмерное движение, просто добавьте уравнение z-движения модели с постоянной скоростью, поскольку z-движение не связано с x-y движением в модели с постоянным поворотом.
Так я собственно  и сделал.

Reply:
Согласен, всё верно, не сразу увидел что ты z,vz, а потом omega